mkdocs.yml

```diff
@@ -17,6 +17,7 @@ nav:
     - 'Flow': 'knowledge/flow.md'
     - 'Cadence': 'knowledge/cadence.md'
     - 'Flare': 'knowledge/flare.md'
+    - 'History Of Scaling in Blockchain': 'knowledge/scalinghistory.md'
```

Comment:
I also think this might be a bit of a long title for the navigation sidebar. WDYT of shortening it, just in the navigation, to something like `Scaling History` or something like that? I think the blockchain aspect is a given, when in the context of our documentation